impeller/toolkit/interop/README.md

```diff
@@ -0,0 +1,94 @@
+# Impeller Standalone SDK
+
+## Goals
+
+This Impeller Standalone API is a single-header C API with no external dependencies.
+
+## Full-featured
+
+The library supports all rendering operations supported by Flutter with few exceptions. An optional text-layout and shaping engine is included by default.
+
+## Easy to Embed
+
+The entire library is distributed as a single dynamic library. The public API can be found in a single C header with no dependencies.
+
+For the common platforms, prebuilt artifacts are generated per Flutter Engine commit.
+
+## Easy Interoperability
+
+The C API allows for explicit management of object lifecycle and is well suited for the generation of automated bindings to languages like Rust, Dart, Lua, etc…
+
+## Lightweight
+
+The core rendering engine is less than 200 KB compressed.
+
+The text layout and shaping engine along with the bundled ICU data tables brings the size up to ~2.5 MB. If the application does not need text layout and shaping, or can interface with an existing library on the target platform, it is recommended to generate the SDK without built-in support for typography. Impeller is a rendering engine but typography is really hard to get right and needed by most users of a rendering engine. That is why one is included by default. But it is optional.
+
+> [!CAUTION]
+> Users of the prebuilt artifacts should strip the binaries before deployment as these contain debug symbols.
+
+## Performant
+
+Built to perform the best when using a modern graphics API like Metal or Vulkan and when running on mobile tiler GPUs like the ones found in smartphones and AppleSilicon/ARM desktops.
```

Comment:
might be worth caveating that libimpeller only expose GLES currently

Reply:
I was going to fix that over the weekend. But yeah, I added a note.